install/eclipse.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Installer for JetBrains Toolbox
+# This will allow the installation, upgrade and management of other JetBrains tools
+# Running the tool itself will automatically add an icon to the launcher
+
```

Comment:
Applied strict mode to all scripts but need to turn it off quite frequently in the sdkman and nvm scripts as they source third-party scripts that don't comply.